### PR TITLE
feat(connector): add request & response logs for top 4 connector

### PR DESCRIPTION
**Pull request by NishantJoshi00**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Adding logs to request and response of connectors: (stripe, checkout, zen, bluesnap)

The concrete solution is implemented in the following.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Comment by ArjunKarthik** [comment]
@NishantJoshi00 We need to capture the error response as well, logging in get_error_response also required

File: crates/router/src/connector/bluesnap.rs [hunk 1]
```diff
@@ -179,6 +180,7 @@ impl
         req: &types::ConnectorCustomerRouterData,
     ) -> CustomResult<Option<String>, errors::ConnectorError> {
         let connector_request = bluesnap::BluesnapCustomerRequest::try_from(req)?;
+        router_env::logger::info!(?connector_request);
```
Comment (ArjunKarthik) [inline crates/router/src/connector/bluesnap.rs:183]: router_env::logger::info!(connector_request=?connector_request);

**Review by ArjunKarthik — APPROVED** [review]

**Review by ArjunKarthik — CHANGES_REQUESTED** [review]
Need changes in logging connector request

File: crates/router/src/connector/bluesnap.rs (hunk 1 above)
Comment (NishantJoshi00) [reply crates/router/src/connector/bluesnap.rs:183]: not required this will actually use the keyword connector_request

**Review by jarnura — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Pull request merged**